lib/charms/vault_k8s/v0/vault_client.py

```diff
@@ -515,68 +576,345 @@ def make_latest_pki_issuer_default(self, mount: str) -> None:
         except (TypeError, KeyError):
             logger.error("Issuers config is not yet created")
 
-    def _get_autounseal_policy_name(self, relation_id: int) -> str:
-        """Return the policy name for the given relation id."""
-        return f"charm-autounseal-{relation_id}"
+    def create_transit_key(self, mount_point: str, key_name: str) -> None:
+        """Create a new key in the transit backend."""
+        response = self._client.secrets.transit.create_key(mount_point=mount_point, name=key_name)
+        logging.debug(f"Created a new transit key. response={response}")
```

Comment:
Let's avoid f-strings in logs.

Reference:
- https://docs.google.com/document/d/1H5W2oi7cuTaz86taTQiITv49jpJNKzzDZ2G25IQpE4w/edit?tab=t.0#heading=h.84ez402ai5cq

Reply:
Fixed in https://github.com/canonical/vault-k8s-operator/commit/a09aefe4129ccad0ad30965b0de2b1320de6a77a